projects/documentation/e2e/published.spec.ts

```diff
@@ -15,7 +15,7 @@ test.describe('search and go', () => {
     const startURL =
         process.env.NODE_ENV === 'CI'
             ? process.env.DOC_PREVIEW_URL
```

Comment:
Should we also check if Storybook has loaded correctly - for example, by verifying that a known selector or a specific story element is present?

Reply:
Happy to defer!